src/main/java/me/itzg/tryetcdworkpart/config/WorkerProperties.java

```diff
@@ -14,7 +14,7 @@
 public class WorkerProperties {
 
   @DurationUnit(ChronoUnit.SECONDS)
-  Duration leaseDuration = Duration.ofSeconds(5);
+  Duration leaseDuration = Duration.ofSeconds(500);
```

Comment:
It looks like 5 seconds is a bit risky since the keep alive "forever" logic doesn't seem to send off the first keepalive until 5 seconds itself:

https://github.com/etcd-io/jetcd/blob/v0.0.2/jetcd-core/src/main/java/com/coreos/jetcd/internal/impl/LeaseImpl.java#L67:28

So a more practical compromise between 5 and 500 could be something like 30 seconds by default.